dev-lang/v8/v8-7.1.302.31.recipe

```diff
@@ -0,0 +1,138 @@
+SUMMARY="Google's open source JavaScript engine"
+DESCRIPTION="V8 is Google's open source JavaScript engine. V8 is written in \
+C++ and is used in Google Chrome, the open source browser from Google. V8 \
+implements ECMAScript as specified in ECMA-262, 5th edition, and runs on \
+Windows (XP or newer), Mac OS X (10.5 or newer), and Linux systems that use \
+IA-32, x64, or ARM processors. V8 can run standalone, or can be embedded into \
+any C++ application."
+HOMEPAGE="https://v8.dev/"
+COPYRIGHT="2006-2018 The V8 Project Authors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/v8/v8/archive/$portVersion.tar.gz"
+SOURCE_DIR="v8-$portVersion"
+CHECKSUM_SHA256="45770770079b30c3229f6f58b1de4d6c953107aa89e72742a6cd224d2a93f1d0"
+srcGitRev2="211b3ed9d0481b4caddbee1322321b86a483ca1f"
+SOURCE_URI_2="git+https://chromium.googlesource.com/chromium/src/base/trace_event/common.git#$srcGitRev2"
```

Comment:
The buildbot is configured to not use `git+` sources. Any GitHub mirrors of these we could use?

If not, we might have to make archives ourselves and put them on the ports-mirror.

Reply:
It should be possible to use a chromium mirror now that all the V8 dependencies have been integrated in the repository due to this commit [1].

However, the catch here is the size. The chromium mirrors on GitHub are 1GB in size and 4GB+ when extracted. This mirror [2] hosts a smaller chromium source size of 500 MB with the same extracted size and much higher download speed (Hosted on Google App Engine). So apart from the size of the source, this mirror can be used to replace the +git sources.   

[1] https://github.com/chromium/chromium/commit/e7001969a4527bc416ef354a78195b942160e79c
[2] https://gsdview.appspot.com/chromium-browser-official/

Reply:
It appears Gentoo does the same: https://data.gpo.zugaina.org/didactic-duck/dev-lang/v8/v8-3.25.9.ebuild

So that sounds fine.